### DISCUSSION TITLE
Retrieving project codes that are lost via AWS

### DISCUSSION THREAD

**Discussion by n0110n01** [mozilla/hubs#5860] [Hubs Cloud Questions]
We recently lost our project folder with all the backups.
Our custom client is already hosted on AWS and live.
Is there any way to get the project files from the AWS?

**Comment by n0110n01** [comment]
Any possible help on this case?

**Reply by pattersonbl2** [reply]
You lose your project folder locally and need to grab the deploy project from aws?   To my knowledge, there is no way to get it back.  You could look in the s3 buckets for your instance to find code elements there.
